www/index.html

```diff
@@ -38,23 +20,154 @@
     -->
     <!-- <meta http-equiv="Content-Security-Policy" content="default-src 'self' data: gap: 'unsafe-inline' https://ssl.gstatic.com; style-src 'self' 'unsafe-inline'; media-src *" /> -->
 
-    <link rel="stylesheet" type="text/css" href="css/index.css" />
-    <title>Hello World</title>
+    <link rel="stylesheet" href="./libs/bootstrap/3.3.7/css/bootstrap.min.css" >
```

Comment:
Why did you choose bootstrap as a CSS framework? What are the pros and cons compared to other solutions out there, or even writing from scratch?